### PR TITLE
INTLY-3980 - new codeready task for image upgrades

### PR DESCRIPTION
**Pull request by tremes**
## Verification Steps
As the verifier of the PR the following process should be done:

### Installation Verification
- Ensure the author of the PR has attached a log of the installation run from his branch to the jira or pr and check that it exited as expected.
- Verify the fresh installation is correct on cluster provided by PR author 
### Upgrade Verification
- After installation verification, notify the PR author to begin an upgrade on their cluster
- Ensure the developer of the PR has attached a log of the upgrade run from his branch to the jira or pr and check that it exited as expected. 
- If possible, look at the tasks that ran and see they match the PR
- Verify the upgrade is correct on cluster provided by PR author 

## Is an upgrade task required and are there additional steps needed to test this?

- [ ] Yes
- [ ] No

- [ ] Tested Installation
- [ ] Tested Uninstallation
- [ ] Tested or Created follow on for Upgrade

### CODE REVIEW

File: roles/codeready/tasks/upgrade.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (pmccarthy) [inline roles/codeready/tasks/upgrade.yml]: Not sure we need an `upgrade.yml` task in this role any longer. Should be ok to just have an `upgrade_images.yml` task that gets called by the main `upgrade.yml` playbook

Comment (tremes) [reply roles/codeready/tasks/upgrade.yml]: OK sounds reasonable. I'll check the main playbook.

Comment (tremes) [reply roles/codeready/tasks/upgrade.yml]: Updated. Can you please check once again @pmccarthy ?

**Comment by tremes** [comment]
Fixed formatting which was broken similarly to https://github.com/integr8ly/installation/pull/1114/f

**Pull request merged**